app/screens/ScorecardDetail/ScorecardDetail.js

```diff
@@ -113,7 +86,7 @@ class ScorecardDetail extends Component {
 
   downloadIndicator = () => {
     if (this.state.isIndicatorDownloaded && this.state.isLanguageIndicatorDownloaded && this.state.isAllAudioDownloaded) {
-      this.updateDownloadProgress(0.6);
+      this.updateDownloadProgress(getEachSectionPercentage() * 3);
```

Comment:
@limkimsan please extract 3 phases into a kind of scorecard setting constant and give comment why 3 phases.